doc/build-unix.md

```diff
@@ -134,9 +134,9 @@ Optional:
 
     sudo dnf install miniupnpc-devel
 
-To build with Qt 5 you need the following:
+To build elements-qt you need the following:
```

Comment:
Hmm, it seems unnecessary to introduce a diff with upstream here, while the original sentence is correct.